tests/unit/filters.spec.ts

```diff
@@ -57,6 +57,19 @@ describe('formatDateTime', () => {
   })
 })
 
+// formatAbsoluteDateTime
+
+describe('formatAbsoluteDateTime', () => {
+  Vue.use(DayJSPlugin)
+  it('renders numbers as date with offset applied', () => {
+    // This should return based on your browsers local time, given unixtime.
+    const d = new Date(0)
+    const offset = d.getTimezoneOffset()
+    const d2 = new Date(d.getTime() + offset * 60)
+    expect(Filters.formatAbsoluteDateTime(d2.getTime())).to.equal('Jan 1, 12:00 AM')
+  })
+})
```

Comment:
It's unclear to me how to test cases like the `todayFormat` vs. `futureFormat` in a deterministic way.  It'd be simple enough to make a naive test, but I'd need to likely mock `Date`  or DayJS in some capacity to ensure the test works even if run at 11:59pm.

If there's some precedent for mocking time (or similar), please let me know!

Reply:
Yep, i've been meaning to implement `mockdate` (search npm for mockdate). For the moment this is fine I think.